types/node/globals.d.ts

```diff
@@ -52,9 +52,9 @@ declare namespace setTimeout {
     function __promisify__(ms: number): Promise<void>;
     function __promisify__<T>(ms: number, value: T): Promise<T>;
 }
-declare function clearTimeout(timeoutId: NodeJS.Timeout): void;
+declare function clearTimeout(timeoutId: NodeJS.Timeout | undefined): void;
```

Comment:
```suggestion
declare function clearTimeout(timeoutId?: NodeJS.Timeout ): void;
```

https://www.typescriptlang.org/play?#code/CYUwxgNghgTiAEAzArgOzAFwJYHtX0hFgAotgAueVZAWwCMQZ4AfeNURLVEYASkoBuOMgG4AUGMIle8APSz4AFQAWWAM7w1ynMgjB4UCGpzwA7jhgBrSRCIxi7EJ258xQA

Reply:
I used `|undefined` as a compromise since that would solve the use case I described. What's the use case for calling `clearTimeout()`?

Reply:
@mohsen1 To clarify: Did you propose to make the argument optional for consistency or is there an actual use case that we should codify in tests?